packages/menu/test/mwc-menu.test.ts

```diff
@@ -299,7 +299,7 @@ suite('mwc-menu', () => {
 
     setup(async () => {
       originalSetTimeout = window.setTimeout;
-      (window as any).setTimeout = (fn: Function) => {
+      (window as any).setTimeout = (fn: () => any) => {
```

Comment:
does `unknown` work here. google style guide prefers over any

Reply:
it'll work but we have to keep the `window as any` cast, i'll update